Sources/PubNub/KMP/KMPPubNub+History.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  KMPPubNub+History.swift
+//
+//  Copyright (c) PubNub Inc.
+//  All rights reserved.
+//
+//  This source code is licensed under the license found in the
+//  LICENSE file in the root directory of this source tree.
+//
+// IMPORTANT NOTE FOR DEVELOPERS USING THIS SDK
+//
+// All public symbols in this file are intended to allow interoperation with Kotlin Multiplatform for other PubNub frameworks.
+// While these symbols are public, they are intended strictly for internal usage.
+//
+// External developers should refrain from directly using these symbols in their code, as their implementation details
+// may change in future versions of the framework, potentially leading to breaking changes.
+
+import Foundation
+
+@objc
+public extension KMPPubNub {
+  func fetchMessages(
```

Comment:
Yes, it works like this. The method you pointed out invokes `pubnub.fetchMessageHistory`. If the caller sets `includeMessageActions` then it will involve `history-with-actions` endpoint

Reply:
Ok. In JS Chat SDK I can see that `includeMeta` and `includeMessageAction` are set to `true` and can't be overridden by the user. 
If the purpose of this wrapper not solely for Chat, then ok ;)